### PR TITLE
Replace unsupported chars, add toLines method, fix empty address fields

### PR DESCRIPTION
**Pull request by 99999989**

### CODE REVIEW

File: infrastructure/src/main/java/ch/admin/seco/alv/service/onlineform/infrastructure/pdf/SupportedCharStripper.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package ch.admin.seco.alv.service.onlineform.infrastructure.pdf;
+
+import java.util.stream.IntStream;
+
+import org.apache.pdfbox.pdmodel.font.encoding.WinAnsiEncoding;
+
+class SupportedCharStripper {
+
+	static String strip(String test) {
```
Comment (pavax) [inline infrastructure/src/main/java/ch/admin/seco/alv/service/onlineform/infrastructure/pdf/SupportedCharStripper.java]: rename test to value

File: infrastructure/src/main/java/ch/admin/seco/alv/service/onlineform/infrastructure/pdf/ProofOfWorkEffortsPdfGenerator.java [hunk 2]
```diff
@@ -180,39 +180,39 @@ private void addTableRow(BaseTable table, List<Integer> cellWidthList, WorkEffor
 			}
 		}
 
-		if (applyChannel.getContactPerson() != null) {
+		if (applyChannel.getContactPerson() != null && !applyChannel.getContactPerson().isEmpty()) {
```
Comment (pavax) [inline infrastructure/src/main/java/ch/admin/seco/alv/service/onlineform/infrastructure/pdf/ProofOfWorkEffortsPdfGenerator.java]: there is also a StringUtils.isEmpty("") that checks both null and empty to simplify these checks

File: infrastructure/src/main/java/ch/admin/seco/alv/service/onlineform/infrastructure/pdf/ProofOfWorkEffortsPdfGenerator.java [hunk 3]
```diff
@@ -180,39 +180,39 @@ private void addTableRow(BaseTable table, List<Integer> cellWidthList, WorkEffor
...
+		if (applyChannel.getPhone() != null && !applyChannel.getPhone().isEmpty()) {
 			applyChannelDetails.add(applyChannel.getPhone());
 		}
 		return applyChannelDetails;
 	}
 
 	private String getAddressLine(ApplyChannelAddress address) {
 		StringBuilder addressLine = new StringBuilder();
-		if (address.getStreet() != null) {
+		if (address.getStreet() != null && !address.getStreet().isEmpty()) {
```
Comment (pavax) [inline infrastructure/src/main/java/ch/admin/seco/alv/service/onlineform/infrastructure/pdf/ProofOfWorkEffortsPdfGenerator.java]: use StringUtils.isEmpty(address.getStreet())

File: infrastructure/src/main/java/ch/admin/seco/alv/service/onlineform/infrastructure/pdf/SupportedCharStripper.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+		if (test == null) {
+			return null;
+		}
+		StringBuilder strippedString = new StringBuilder();
+		IntStream.range(0, test.length()).forEach(i -> {
```
Comment (pavax) [inline infrastructure/src/main/java/ch/admin/seco/alv/service/onlineform/infrastructure/pdf/SupportedCharStripper.java]: I guess you could also just use IntStream intStream = test.chars();

Comment (99999989) [reply infrastructure/src/main/java/ch/admin/seco/alv/service/onlineform/infrastructure/pdf/SupportedCharStripper.java]: Thx for the hint. Done.

File: infrastructure/src/main/java/ch/admin/seco/alv/service/onlineform/infrastructure/pdf/ProofOfWorkEffortsPdfGenerator.java (hunk 3 above)
Comment (99999989) [reply infrastructure/src/main/java/ch/admin/seco/alv/service/onlineform/infrastructure/pdf/ProofOfWorkEffortsPdfGenerator.java]: done

File: infrastructure/src/main/java/ch/admin/seco/alv/service/onlineform/infrastructure/pdf/ProofOfWorkEffortsPdfGenerator.java (hunk 2 above)
Comment (99999989) [reply infrastructure/src/main/java/ch/admin/seco/alv/service/onlineform/infrastructure/pdf/ProofOfWorkEffortsPdfGenerator.java]: done

File: infrastructure/src/main/java/ch/admin/seco/alv/service/onlineform/infrastructure/pdf/SupportedCharStripper.java (hunk 1 above)
Comment (99999989) [reply infrastructure/src/main/java/ch/admin/seco/alv/service/onlineform/infrastructure/pdf/SupportedCharStripper.java]: done

**Pull request merged**